lib/stripe/api_resource.rb

```diff
@@ -2,6 +2,13 @@ module Stripe
   class APIResource < StripeObject
     include Stripe::APIOperations::Request
 
+    # A flag that can be set a behavior that will cause this resource to be
+    # encoded and sent up along with an update of its parent resource. This is
+    # usually not desirable because resources are updated individually on their
+    # own endpoints, but there are certain cases, changing a customer's default
```

Comment:
Ohh right. So as I understand it, using the `customer.source=` accessor will actually replace the default with the new source (as opposed to something like `customer.sources.create` which adds a new one).

True though that this isn't really a need-to-know here though. Let me rephrase this a bit to be more apparent.